igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -91,18 +93,54 @@
     "hyp3 = sdk.HyP3(prompt=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "f387a5e2-f145-41b8-8cb8-f8d29155fff0",
+   "metadata": {},
+   "source": [
+    "2. Select your data: Use ASF's [Vertex](https://search.asf.alaska.edu/) data search portal to find Sentinel-1 scenes your interesed in processing."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "739d2ebe-e14e-4998-a37e-f9b32d106902",
    "metadata": {},
    "outputs": [],
    "source": [
-    "project_name = 'edgecumbe_example'\n",
     "reference = 'S1B_IW_SLC__1SDV_20180628T151540_20180628T151607_011575_015476_4673'\n",
     "secondary = 'S1B_IW_SLC__1SDV_20190705T151547_20190705T151614_017000_01FFC4_D62F'"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d74b4574-0bcd-4ce7-8bda-cc2bb95f17ee",
+   "metadata": {},
+   "source": [
+    "3. Define your processing parameters: Determine the specific processing parameters for your InSAR product. This includes selecting the desired satellite data acquisitions, multilook, and whether to apply a water mask before unwrapping. You can read more about the available parameters within the [HyP3 SDK documentation](https://hyp3-docs.asf.alaska.edu/using/sdk_api/#hyp3_sdk.hyp3.HyP3.submit_insar_job)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3cfa37d6-ec58-4209-bd7d-43b42fd3c2a6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "looks='20x4'\n",
+    "include_dem=True\n",
+    "include_look_vectors=True\n",
+    "apply_water_mask=True"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "427de656-ef70-45d1-b0be-ee63375182ad",
+   "metadata": {},
+   "source": [
+    "4. Submit a processing request: Once you have defined your parameters, use the HyP3 SDK's [`submit_insar_job`](https://hyp3-docs.asf.alaska.edu/using/sdk_api/#hyp3_sdk.hyp3.HyP3.submit_insar_job) function to submit a processing request. You can use the `project_name` name argument to group sets of requests together under one name so that you can easily look them up later."
```

Comment:
```suggestion
    "4. **Submit a processing request**: Once you have defined your parameters, use the HyP3 SDK's [`submit_insar_job`](https://hyp3-docs.asf.alaska.edu/using/sdk_api/#hyp3_sdk.hyp3.HyP3.submit_insar_job) function to submit a processing request. You can use the `project_name` name argument to group sets of requests together under one name so that you can easily look them up later."
```